packages/code-du-travail-modeles/src/modeles/conventions/1516_organismes_formation.yaml

```diff
@@ -0,0 +1,66 @@
+contrat salarié . convention collective . organismes de formation:
+  description: Convention collective nationale des organismes de formation du 10 juin 1988.
+  applicable si: convention collective = 'IDCC1516'
+  valeur: oui
+  cdtn:
+    idcc: 1516
+    préavis-retraite: true
+
+contrat salarié . convention collective . organismes de formation . départ à la retraite:
+  applicable si: départ à la retraite
+  remplace: préavis de retraite collective
+  grille:
+    assiette: ancienneté
+    tranches:
+      - montant: 0 mois
+        plafond: 6 mois
+      - montant: 1 mois
+        plafond: 24 mois
```

Comment:
ici c'est 25

Reply:
Il est noté `moins de 2 ans` et `2 ans et plus` dans le Excel, donc ce doit être 24 mois à mon avis.
<img width="1726" alt="image" src="https://user-images.githubusercontent.com/992514/144275513-63d3e781-2c83-4656-96d5-287bea82a055.png">

Reply:
Pardon le comment est sur la mauvaise ligne il devrait etre ici : https://github.com/SocialGouv/code-du-travail-numerique/pull/4056/files/c0c65150d524e57a76c2c79c996109db68d607e1#diff-c8589deb5f9413211614019266529fe9c0697bdceae2c874786154f6b117b6baR43